packages/node_modules/@webex/plugin-meetings/webex.js

```diff
@@ -12,6 +12,7 @@ require('@webex/internal-plugin-support');
 require('@webex/plugin-logger');
 require('@webex/plugin-meetings');
 require('@webex/plugin-device-manager');
+require('webrtc-adapter');
```

Comment:
I don't see this included anywhere else in the PR. Why do we need to bring it in now?

Reply:
webrtc-adapter is a shim for webrtc peerconnection which fixes any differences in the API in browser i did not use it before but as we are seeing difference in the safari behavior on ontrack , once i added it got fixed for some reason so i left it ,

lock file did not get updated , i assume that because the phone plugin used to use it so its there